tests/vmi_networking_test.go

```diff
@@ -826,6 +826,8 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				if !tests.HasLiveMigration() {
 					Skip("LiveMigration feature gate is not enabled in kubevirt-config")
 				}
+
+				libnet.SkipWhenClusterHasASingleNode(virtClient)
```

Comment:
The separation of commits is a bit odd.
I would expect that the changes introduces are exercised in the same commit to show it works.

So, how about this split:
- First commit to export the skip to a central location, replacing all existing calls to the new implementation.
- Second commit to add the new skip and explain why it is needed. (it actually is missing the reasoning)

Reply:
No longer relevant.